src/cephadm/cephadm.py

```diff
@@ -1577,10 +1577,10 @@ def run(self) -> None:
                                               data=data,
                                               endpoint='/data',
                                               ssl_ctx=self.ssl_ctx)
-                response_json = json.loads(response)
                 if status != 200:
                     logger.error(f'HTTP error {status} while querying agent endpoint: {response}')
                     raise RuntimeError
```

Comment:
the `URLError` exception context is gone, but add the response/reason to the `RuntimeError` anyway?

Reply:
Sure, I guess the RuntimeError could just include the same things the error log statement above it has (status code and response string).

Reply:
added that in